README.md

```diff
@@ -38,33 +38,39 @@ The AC calculations are based on full Newton-Raphson algorithm. The DC calculati
  - a distributed slack (on generation or on loads or on conform loads); Slack bus selection could be automatic or explicit as explained [here](https://www.powsybl.org/pages/documentation/simulation/powerflow/openlf.html#parameters).
  - a support of generators' active and reactive power limits, included the support of reactive capability curves.
  - 5 starting point modes: flat, warm, only voltage angles initialization based on a DC load flow, only voltages magnitude initialization based on a specific initializer, or both voltages angle and magnitude initialization based on the two previous methods.
- - a support of non impedant branches, including complex non impedant sub-networks.
+ - a support of zero impedance branches, including complex zero impedance subnetworks, particularly important in case of voltage controls and topology changes involved in contingencies or in remedial actions.
  - a multiple synchronous component calculation, generally linked to HVDC lines.
+ - A modeling of secondary voltage control following research of [Balthazar Donon, Liège University](https://www.montefiore.uliege.be/cms/c_3482915/en/montefiore-directory?uid=u239564).
+ - Support of asymmetrical calculations.
 
```

Comment:
Maybe a word on multiple state vector update methods in the Newton Raphson algorithm?
Something like: "- An implementation of three methods to update the state vector in the Newton Raphson algorithm: classic, rescaling under maximum voltage change and linear search rescaling."